Resources/Textures/Nuclear14/Objects/Weapons/Guns/Rifles/Assault Carbine.rsi/meta.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "i dont have a single idea of what put in here. im supposed to just delete it?",
```

Comment:
If you made the sprites entirely yourself you can put whatever you like in here. Most people put something like "Made for Nuclear 14 by #insert github or discord handle here#". If the assets are ever from SS13 you can put the codebase and commit link here.